src/com/github/arseniydubrovski/ReversePolishNotation.java

```diff
@@ -0,0 +1,101 @@
+package com.github.arseniydubrovski;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+import java.util.HashMap;
+
+/**
+ * realize  Reverse Polish Notation.
+ */
+class ReversePolishNotation {
+
+  private static final HashMap<Character, Integer> PRIORITIES = new HashMap<>();
+  private static final int LOW_PRIORITY = 1;
+  private static final int HIGH_PRIORITY = 2;
+
+  static {
+    PRIORITIES.put('+', LOW_PRIORITY);
+    PRIORITIES.put('-', LOW_PRIORITY);
+    PRIORITIES.put('*', HIGH_PRIORITY);
+    PRIORITIES.put('/', HIGH_PRIORITY);
+  }
+
+  private char[] inputString;
+
+  /**
+   * realize  Reverse Polish Notation.
+   *
+   * @param inputString string with arithmetic expression.
+   */
+  public ReversePolishNotation(char[] inputString) {
+    this.inputString = inputString;
+  }
+
+
+  /**
+   * solve arithmetic expression by Reverse Polish Notation.
+   *
+   * @return solution of arithmetic expression
+   */
+  public Integer reversePolishNotation() {
```

Comment:
If it solves something, why it's called reversePolishNotation? Find better name for this method